tests/testthat/test-extract_variable.R

```diff
@@ -13,6 +13,19 @@ test_that("extract_variable works the same for different formats", {
   draws_matrix <- as_draws_matrix(example_draws())
   mu_matrix <- extract_variable(draws_matrix, "mu")
   expect_equal(as.vector(mu_matrix), as.vector(mu_array))
+
+  draws_rvars <- as_draws_rvars(example_draws())
+  mu_matrix <- extract_variable(draws_rvars, "mu")
+  expect_equal(as.vector(mu_matrix), as.vector(mu_array))
+})
+
+test_that("extract_variable works for draws_rvars on an indexed variable", {
```

Comment:
do not forget to add a test once the bug I noted for `extract_variable.draws_rvars` has been fixed.

Reply:
done